packages/paste-website/src/components/site-wrapper/sidebar/SidebarNavigation.styles.ts

```diff
@@ -17,86 +17,117 @@ interface SiteNavNestListProps {
 }
 
 export const SiteNavNestList = styled(SiteNavList)<SiteNavNestListProps>`
-  display: ${props => (props.isOpen ? 'block' : 'none')};
   text-transform: capitalize;
-  background-color: ${themeGet('colors.colorGray30')};
-  overflow: hidden;
-  border-bottom-right-radius: ${themeGet('radii.borderRadius20')};
-  border-bottom-left-radius: ${themeGet('radii.borderRadius20')};
-  padding-left: ${themeGet('space.space40')};
 `;
 
 export const SiteNavItem = styled.li`
-  margin: ${themeGet('space.space10')} 0;
+  margin: ${themeGet('space.space20')};
   font-size: ${themeGet('fontSizes.fontSize30')};
   line-height: ${themeGet('lineHeights.lineHeight20')};
 `;
 
+export const SiteNavNestedItem = styled(SiteNavItem)`
```

Comment:
Any scope to convert a bunch of this to `Box`?

Reply:
Yeah. I refactored it all to use `Box`.